content/awell-orchestration/api-reference/queries/get-published-pathways.mdx

```diff
@@ -16,7 +16,7 @@ query GetPublishedPathwayDefinitions {
       id
       title
       version
-      dataPointDefinitions {
+      data_point_definitions {
```

Comment:
@rahulkeerthi Is this a breaking change or was the documentation incorrect?

Reply:
no it is not, you can use either field